django/contrib/admin/templates/admin/change_list.html

```diff
@@ -60,8 +60,8 @@
         {% if cl.has_filters %}
           <div id="changelist-filter">
             <h2>{% translate 'Filter' %}</h2>
-            {% if cl.preserved_filters %}<h3 id="changelist-filter-clear">
-              <a href="?{% if cl.is_popup %}_popup=1{% endif %}">&#10006; {% translate "Clear all filters" %}</a>
+            {% if cl.get_filters_params %}<h3 id="changelist-filter-clear">
```

Comment:
This will display "Clear all filters" for other parameters passed via the query string, e.g.
`http://localhost:8000/admin/auth/user/?username__startswith=l`:

![obraz](https://user-images.githubusercontent.com/2865885/82284795-21d76900-999a-11ea-9488-8dccb5141bb4.png)